### PR TITLE
feat: Add user agent header

### PR DESCRIPTION
**Pull request by agusaldasoro**
### Changes
- Update `catalyst-commons` library to latest: `2.0.0`
- Configure `Fetcher` to send the header `user_agent` in `CatalystClient`, `ContentClient` & `LambdaClient` with the information of the service running.

### CODE REVIEW

File: src/CatalystClient.ts [hunk 1]
```diff
@@ -24,21 +23,30 @@ import { sanitizeUrl } from './utils/Helper'
 import { ContentClient, DeploymentOptions } from './ContentClient'
 import { LambdasClient } from './LambdasClient'
 import { DeploymentWithMetadataContentAndPointers } from './ContentAPI'
+import { RequestOptions } from 'dcl-catalyst-commons/dist/utils/FetcherConfiguration'
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { version: VERSION } = require('../package.json')
```
Comment (nchamo) [inline src/CatalystClient.ts:28]: Could this be changed to? (and same with the ones below)
```import { version: VERSION } from '../package.json'```

Comment (nchamo) [reply src/CatalystClient.ts:28]: Also, what value does this have on tests?

File: src/ContentAPI.ts [hunk 2]
```diff
@@ -20,6 +19,7 @@ import {
 import { Readable } from 'stream'
 import { DeploymentData } from './utils/DeploymentBuilder'
 import { DeploymentOptions } from './ContentClient'
+import { RequestOptions } from 'dcl-catalyst-commons/dist/utils/FetcherConfiguration'
```
Comment (nchamo) [inline src/ContentAPI.ts:22]: This is not ok. We probably didn't export `FetcherConfiguration` correctly on `dcl-catalyst-commons` (same on the ones below)

**Review by nchamo — COMMENTED** [review]
Some comments

File: src/CatalystClient.ts (hunk 1 above)
Comment (agusaldasoro) [reply src/CatalystClient.ts:28]: I've only added tests for `ContentClient`, and the value of the version is `0.0.0-development`

**Review by nchamo — APPROVED** [review]
Great work!

**Pull request merged**